README.md

```diff
@@ -19,24 +24,46 @@
 
 # 📖 [Portfolio] <a name="about-project"></a>
 
-> This project was created with media query and responsive web design
+> This project was created to design a portfolio webpage
+> It is built with flexbox, css grid,  media query for responsive web design.
 
-**[Portfolio]** is a webpage with toolbar and headline section.
+**[Portfolio]** is a webpage with toolbar, headline section, project section, about section and contact section.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
 > The tech stack used are html and css.
 
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
```

Comment:
- [ OPTIONAL ] I think it is better if you remove this line that was instructing you to add a link to your deployed project.

Reply:
Thank you sir